packages/web/vite.config.ts.timestamp-1728221477604-4ec75ba1037b3.mjs

```diff
@@ -0,0 +1,16 @@
+// vite.config.ts
```

Comment:
This file doesn't look like it should be checked in. Can we remove.

Reply:
Sure